CHANGELOG.md

```diff
@@ -22,7 +23,8 @@
       * Add `approveOrder(request:completion:(CardResult?, Error?) -> Void)` to `CardClient`
       * Add `vault(request:completion:(CardVaultResult?, Error?) -> Void)` to `CardClient`
       * Add `approveOrder(request:) async throws -> CardResult`
-      * Add `vault(vaultRequest:) async throws -> CardVaultResult`      
+      * Add `vault(vaultRequest:) async throws -> CardVaultResult`   
+      * Add `.threeDSecureCanceled` to `CardClientError`   
```

Comment:
Nit, these are a rename vs an add. In case a merchant expects the old ones to still be there

Reply:
I wanted to show that a new async/await function was added. 
I think rename might confuse them because it really is a new function.